pkg/scanners/trivy/trivy.go

```diff
@@ -49,6 +49,7 @@ var (
 	vulnDBRepository       = flag.String("db-repository", "ghcr.io/aquasecurity/trivy-db", "vulnerability database repository")
 	rekorURL               = flag.String("rekor-url", "https://rekor.sigstore.dev", "Rekor URL")
 	deleteScanFailedImages = flag.Bool("delete-scan-failed-images", true, "whether or not to delete images for which scanning has failed")
+	imageScanTimeout       = flag.Duration("image-scan-timeout", time.Hour, "Duration for timeout for images scanned. Default unit is ns.")
```

Comment:
Be clear that this is a timeout for the entire scan process, not the timeout for an individual image

Reply:
this one was for per image, but it should be clarified now from the total run timeout